conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -44,23 +38,17 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # Cywing's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
-    - cmd: rmdir C:\cygwin /s /q
-    - appveyor DownloadFile "https://raw.githubusercontent.com/pelson/Obvious-CI/master/bootstrap-obvious-ci-and-miniconda.py"
-    - cmd: python bootstrap-obvious-ci-and-miniconda.py %CONDA_INSTALL_LOCN% %TARGET_ARCH% %CONDA_PY:~0,1% --without-obvci
-    - cmd: set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda config --set show_channel_urls true
-    - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
+    - cmd: if %TARGET_ARCH%==x64 (set miniconda_suffix="-x64") else (set miniconda_suffix="")
+    - cmd: set PATH=C:\Miniconda3%miniconda_suffix%;C:\Miniconda3%miniconda_suffix%\Scripts;%PATH%
+
     {%- for channel in channels.get('sources', []) %}
     - cmd: conda config --add channels {{ channel }}{% endfor %}
+    - cmd: conda update --all --quiet --yes
+    - cmd: conda install --quiet --yes conda-forge-build-setup
+    - cmd: activate root
```

Comment:
This doesn't work on my windows miniconda install: 

```
C:\data\downloads\inkscape
λ activate root
No environment named "root" exists in C:\portabel\miniconda\envs, or is not a valid conda installation directory.
```

I've actually no idea how to activate the root environment